src/components/Page/Base.js

```diff
@@ -25,6 +27,15 @@ function BasePage({
 
   return (
     <div className={classes.root}>
+      <Script id="google-analytics">
```

Comment:
We should be using [`afterInteractive`](https://nextjs.org/docs/basic-features/script#afterinteractive) strategy unless you've encountered issues with it.